MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -222,11 +222,7 @@ namespace WPEFramework {
 
             LOGINFO("Reboot_Pending :%s",g_is_reboot_pending.c_str());
 
-#if defined (SKY_BUILD)
-            bool internetConnectStatus = true;
-#else
             bool internetConnectStatus = isDeviceOnline();
```

Comment:
Pls increase timeout for isDeviceOnline to 2 min (4 checks every 30 seconds)

Reply:
added the retry logic.